docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -405,11 +408,26 @@ endif::[]
 EOF
 ----
 +
-* Prepare adopted EDPM workloads to use Ceph backend for Block Storage service (cinder), if configured so
+<1> If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
+<2> If using a custom DNS Domain, modify the `spec:nodes:[NODE NAME]:hostName` to use fqdn for the node.
+<3> Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+
+. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the Compute nodes before adoption. This configuration is stored in the `external_ids` column in the `Open_vSwitch` table in the Open vSwitch database:
```

Comment:
Maybe drop "that are configured" and "as were set" for fluff and passivity
Also:
"Ensure that the `ovn-controller` settings in the `OpenStackDataPlaneNodeSet` CR are the same settings as the Compute nodes before adoption."
or 
"Ensure that you use the same `ovn-controller` settings in the `OpenStackDataPlaneNodeSet` CR that you used in the Compute nodes before adoption."

to make it more user focused.